src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -483,7 +483,7 @@ private function parseFileToDOM(string $file): \DOMDocument
                 }
             }
             if ($errors) {
-                throw new InvalidArgumentException(\sprintf('Unable to parse file "%s": ', $file).implode("/n", $errors), $e->getCode(), $e);
```

Comment:
Shouldn’t this have been `\n` instead?

Reply:
I honestly have no clue. I assume there is some test validating result of current behaviour. If you eager to investigate/fix this, please raise it separately.
Unfortunately, I won't be able to check this on my side shortly

Reply:
* https://github.com/symfony/symfony/pull/58487